### PR TITLE
Fix application of Transform passes in routing_example.ipynb

### PR DESCRIPTION
**Pull request by sjdilkes**
`Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(circuit)` only applies the `Transform.DecomposeBRIDGE(circuit)` pass. Notebook is changed to apply each pass separately.

### CODE REVIEW

File: examples/routing_example.ipynb [hunk 1]
```diff
@@ -1058,9 +1355,10 @@
     "graph_routed_circuit = route(gp_ex_circ, ourense_device)\n",
     "noise_aware_routed_circuit = route(np_ex_circ, ourense_device)\n",
     "\n",
-    "Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(line_routed_circuit)\n",
-    "Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(graph_routed_circuit)\n",
-    "Transform.RemoveRedundancies().DecomposeSWAPtoCX().DecomposeBRIDGE().apply(noise_aware_routed_circuit)\n",
+    "\n",
+    "for c in [line_routed_circuit, graph_routed_circuit, noise_aware_routed_circuit]:\n",
+    "    Transform.DecomposeBRIDGE().apply(c)\n",
+    "    Transform.DecomposeSWAPtoCX().apply(c)\n",
```
Comment (cqc-alec) [inline examples/routing_example.ipynb:1361]: No `RemoveRedundancies`?

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: examples/routing_example.ipynb (hunk 1 above)
Comment (cqc-alec) [reply examples/routing_example.ipynb:1361]: We could take the opportunity to illustrate the `SequencePass` construction (making a `SequencePass` once and then applying it to each circuit).

File: examples/routing_example.ipynb [hunk 2]
```diff
@@ -321,19 +367,10 @@
...
-     },
-     "metadata": {},
-     "execution_count": 11
-    }
-   ],
+   "outputs": [],
    "source": [
     "from qiskit import IBMQ\n",
-    "IBMQ.load_account()"
+    "IBMQ.load_account();"
```
[the remark this answers is not in the document]
Comment (sjdilkes) [reply examples/routing_example.ipynb:373]: Oops - I was trying to nicely suppress an IBMQ UserWarning and left this in by a mistake after suppressing it another way.

File: examples/routing_example.ipynb (hunk 1 above)
Comment (sjdilkes) [reply examples/routing_example.ipynb:1361]: `SequencePass ` along with `CompilationUnit` is introduced at the end of the notebook, so while I considered this I decided it was better to first introduce them together at the end. 

Regarding removing `RemoveRedundancies`, I thought that it was unnecessary and potentially overcomplicated that example at that point, so it was a deliberate decision.

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**